config/config.go

```diff
@@ -306,28 +306,28 @@ func New(logger *zap.SugaredLogger, osArgs []string) (config, error) {
 
 	mainFS.StringVar(&cfg.Controller.SafeMode.Environment, "safemode-environment", "", "Specify the 'location' this controller is run in. All disruptions must have an annotation of chaos.datadoghq.com/environment configured with this location to be allowed to create")
 
-	if err := viper.BindPFlag("conroller.safemode.environment", mainFS.Lookup("safemode-environment")); err != nil {
+	if err := viper.BindPFlag("controller.safeMode.environment", mainFS.Lookup("safemode-environment")); err != nil {
```

Comment:
tbh this type of change scares me 😅

Reply:
does it really have an impact? I thought it would just lower case it internally to avoid such conflicts

Reply:
It does not actually have any impact, but camelCase is what we use for the json tags and in the configmap, so we should be consistent here